### DISCUSSION TITLE
displayinig regular mesh instead of voxels

### DISCUSSION THREAD

**Discussion by cuix-github** [ProjectPhysX/FluidX3D#107] [General]
Hi there,

I have been playing this project for a while. But I didn't figure out a way to display actual meshes instead of voxels in the INTERACTIVE_GRAPHIS mode. Is there an implementation for that?

**Comment by ProjectPhysX** [comment]
Hi @cuix-github, I have not implemented rendering of the input stl mesh. This is to reduce VRAM footprint by immediately freeing VRAM allocated for the mesh (components) after voxelization, and to avoid the false impression of a smooth surface when in the simulation model the surface is always voxelized.

**Reply by cuix-github** [reply]
Make sense! Thanks for the explanation. Sad my laptop only has 4GB (4000u) VRAM for the sim and the voxels give a look like coarse LEGO :-(
